hic_qc.py

```diff
@@ -137,7 +137,7 @@ def __init__(self, outfile_prefix='Read_mate_dist', sample_type='genome', thresh
                                'perc_pairs_greater_10k_on_contigs_greater_10k': ('pairs_greater_10k_on_contigs_greater_10k', 'pairs_on_contigs_greater_10k'),
                                'perc_pairs_intra_hq_gt10kbp': ('pairs_intracontig_hq_gt10kbp', 'pairs_intracontig_hq'),
                                'perc_pairs_on_same_strand': ('pairs_on_same_strand', 'total_pairs_on_same_contig'),
-                               'perc_pairs_on_same_strand_hq': ('pairs_on_same_strand_hq', 'pairs_intracontig_hq'),
+                               'perc_pairs_on_same_strand_hq': ('pairs_on_same_strand_hq', 'total_read_pairs_hq'),
```

Comment:
I don't think I agree with this change. It makes the same strand stat less useful as a measure of HiC to shotgun read ratio, and makes it dependent on assembly contiguity. For fragmented assemblies, it will underestimate true same strand rate.

Reply:
Just resolving comment: subsequent discussion led to a redesign, which was implemented